CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# BOX-STORAGE-VERSION TBD
+
+## Changed
+
+- `class StateSchema`'s method `undictify()` now returns a `StateSchema` object instead of a python `dict`
```

Comment:
It looks like this file might be autogenerated and might cause conflicts - just a note to take a look later